reporter.go

```diff
@@ -234,7 +235,7 @@ func (r *remoteReporter) processQueue() {
 				atomic.AddInt64(&r.queueLength, -1)
 				if flushed, err := r.sender.Append(span); err != nil {
 					r.metrics.ReporterFailure.Inc(int64(flushed))
-					r.logger.Error(err.Error())
+					r.logger.Error(fmt.Sprintf("error reporting span %q: %s", span.OperationName(), err.Error()))
```

Comment:
> `2017/09/08 15:53:50 ERROR: error reporting span "CassandraStore.SearchTable": Span is too large`

Where do the quotes come from?  I was suggesting enclosing `%q` in single-quotes: `'%q'`

Reply:
`%q` prints the string quoted.
see https://play.golang.org/p/YCWkpIgTtP

Reply:
ha, weird....